recipe_scrapers/rewe.py

```diff
@@ -48,13 +48,15 @@ def ingredients(self):
         ingredients = []
 
         for item in list_items:
-            text = item.find("div", {"class": "ld-rds flex-[1_0_0%]"})
-            ingredient_text = (
-                text.get_text(" ", strip=True).replace("\n", "").replace("  ", " ")
+            amount = normalize_string(
+                item.find("div", class_="formattedAmountDiv").get_text()
+            )
+            ingredient = item.find(
+                "span", {"class": "ld-rds break-words leading-6"}
+            ).get_text(" ", strip=True)
+            ingredients.append(
+                normalize_string(f"{amount + " " if amount != "0" else ""}{ingredient}")
```

Comment:
seems like the double quotes are breaking the f-string syntax in older python <= 3.11 versions.

Can you please swap to `normalize_string(f"{amount + ' ' if amount != '0' else ''}{ingredient}")` for example

After this change, if CI passes, we can merge.